db/migrate/20151220030058_add_estimated_dob_to_students.rb

```diff
@@ -0,0 +1,7 @@
+class AddEstimatedDobToStudents < ActiveRecord::Migration
+  def change
+    add_column :students, :estimated_dob, :boolean
+    change_column :students, :estimated_dob, :boolean, null: false #This is because of an SQLite bug which won't lett us add a column with a NOT NULL constraint
```

Comment:
Not really, if it can't be null - then it has to be something - so it makes sense to have a default!